application/Commands/Layouts/LayoutsCommands.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Please rename this file to `LayoutsCommand.php` so that its name matches the class name.